pkg/fission-cli/flag/key/key.go

```diff
@@ -131,6 +131,8 @@ const (
 	EnvImagePullSecret = "imagepullsecret"
 	EnvExecutorType    = "executortype"
 	EnvForce           = force
+	EnvBuilder         = "builder-env"
```

Comment:
We mostlly have CLI arguments without `-` but I think this makes sense for the env argument.